x-pack/packetbeat/magefile.go

```diff
@@ -0,0 +1,112 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build mage
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/magefile/mage/mg"
+
+	devtools "github.com/elastic/beats/v7/dev-tools/mage"
+	packetbeat "github.com/elastic/beats/v7/packetbeat/scripts/mage"
+
+	// mage:import
+	"github.com/elastic/beats/v7/dev-tools/mage/target/common"
+	// mage:import
+	_ "github.com/elastic/beats/v7/dev-tools/mage/target/compose"
+	// mage:import
+	_ "github.com/elastic/beats/v7/dev-tools/mage/target/unittest"
+	// mage:import
+)
+
+func init() {
+	common.RegisterCheckDeps(Update)
+
+	devtools.BeatDescription = "Packetbeat analyzes network traffic and sends the data to Elasticsearch."
+	devtools.BeatLicense = "Elastic License"
+}
+
+// Update updates the generated files.
+func Update() {
+	mg.SerialDeps(packetbeat.FieldsYML, Dashboards, Config)
+}
+
+// Config generates the config files.
+func Config() error {
+	return devtools.Config(devtools.AllConfigTypes, packetbeat.ConfigFileParams(), ".")
+}
+
+// Dashboards packages kibana dashboards
+func Dashboards() error {
+	return devtools.KibanaDashboards("protos")
```

Comment:
```suggestion
	return devtools.KibanaDashboards(devtools.OSSBeatDir("protos"))
```

In x-pack we want to include the OSS dashboards and the x-pack dashboards (but there are none of those now).

Reply:
As it turns out, none of the dashboards are actually in the `beats/packatebeat/protos/**`, but this is still the right way to go.